ci/yaml_to_json.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python
+
+import yaml
+import json
+import os
+import fnmatch
+from collections import OrderedDict 
+
+# directory to store assets for test or release
+assets_dir = os.getcwd() + "/ci/assets"
+
+for file in os.listdir(assets_dir):
+  if fnmatch.fnmatch(file, '*index.yaml'):
+    with open(assets_dir + "/" + file, 'r') as s, open(assets_dir + "/" + os.path.splitext(file)[0] + ".json", "w") as o:
+        try:
+            doc = yaml.safe_load(s)
+            i = 0
+
+            if (doc['stacks'] != None):
+                for item in doc['stacks']:
+                    res = (OrderedDict([
+                        ("displayName", item['name']),
+                        ("description", item['description']),
+                        ("language", item['id']),
+                        ("projectType", "appsodyExtension"),
+                        ("projectStyle", "Appsody"),
+                        ("location", item['templates'][0]['url']),
```

Comment:
I believe you should pull in url for the default-template here

Reply:
Yes, I have fixed this in the latest commit